src/tile.cpp

```diff
@@ -933,7 +933,7 @@ void Tile::addThing(int32_t, Thing* thing)
 
 			items = makeItemList();
 			items->insert(items->getBeginDownItem(), item);
-			++items->downItemCount;
+			items->addDownItemCount(1);
```

Comment:
Yes, but what's the advantage of having a new function when ++ and -- already exist?